File: 프로그래머스/lv0/181906. 접두사인지 확인하기/접두사인지 확인하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string my_string, string is_prefix) {
    int answer = 0;
    string s = "";
    if(my_string.length() > is_prefix.length())
    {
        s = my_string.substr(0, is_prefix.length());
    }
    if(s == is_prefix)
        return 1;
    return answer;
}

File: 프로그래머스/lv0/181906. 접두사인지 확인하기/접두사인지 확인하기.cs
using System;

public class Solution {
    public int solution(string my_string, string is_prefix) {
        int answer = 0;
        string s = "";
        if(my_string.Length > is_prefix.Length)
        {
            s = my_string.Substring(0, is_prefix.Length);    
        }
        Console.Write(s);
        
        if(s == is_prefix)
            return 1;
        return answer;
    }
}